packages/shared/src/utils/errors.ts

```diff
@@ -4,6 +4,11 @@ export function isUnauthorizedError(e: any): boolean {
   return code === 'authentication_invalid' && status === 401;
 }
 
+export function is400Error(e: any): boolean {
```

Comment:
```suggestion
export function is4xxError(e: any): boolean {
```

Reply:
I did it in the commit messages and not here smh